TranslationBackend/app/Http/Controllers/UploadDataController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests;
+use Illuminate\Support\Facades\Storage;
+
+class UploadDataController extends Controller
+{
+    public function index() {
```

Comment:
What's this for?

Reply:
This is used to display a dummy page so that we can test to upload files and update the database in the backend.